lib/class/class_fpas_place.dart

```diff
@@ -245,4 +245,52 @@ class FPASPlace extends Place {
       }
     }
   }
+
+  /// Fetch the server settings from the given FPAS server. This sets in the
+  /// preferences the link to the privacy police, the terms of use and
+  /// the faps server url
+  ///
+  /// [newUrl] the url of the new FAPS server
+  ///
+  /// returns true if the data was successfully fetched,
+  /// returns false if the url is not a valid FPAS server url or something
+  /// else went wrong
+  static Future<bool> fetchServerSettings(String newUrl) async {
+    try {
+      debugPrint(newUrl);
+      Uri newFPASsUri = Uri.parse("$newUrl/sources/server_status");
+      Response response = await http.get(
+        newFPASsUri,
+        headers: {
+          "Content-Type": "application/json",
+          'user-agent': userPreferences.httpUserAgent
+        },
+      );
+      if (response.statusCode != 200) {
+        throw Exception("FPAS Server not reachable");
+      }
+      dynamic data = jsonDecode(utf8.decode(response.bodyBytes));
```

Comment:
In general it's better to type your variables. I haven't found an API doc that describes this particular endpoint but in general the expected type from a JSON endpoint would be `Map<String, dynamic>` (although if you know the values something more precise than `dynamic` there would be preferred).